uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/absence-tab/components/annual-entitlements.component.spec.js

```diff
@@ -118,6 +119,17 @@ define([
       });
     });
 
+    describe('when user wants to see a comment to an entitlement', function () {
```

Comment:
The only way to avoid this confusion is to wrap the notification call into a separate function, but in this case the test will lose its purpose, since I want to be sure that a **notification** is shown, not a wrapper to a notification is called.